platforms/platforms_test.go

```diff
@@ -364,3 +365,83 @@ func TestParseSelectorInvalid(t *testing.T) {
 		})
 	}
 }
+
+func TestNormalize(t *testing.T) {
+	cases := []struct {
+		inputs   []specs.Platform
+		expected specs.Platform
+	}{
+		{
+			// Only OS is filled, but why?
+			inputs: []specs.Platform{
+				{OS: "", Architecture: ""},
+			},
+			expected: specs.Platform{OS: runtime.GOOS, Architecture: ""},
+		},
+		{
+			// Okay to have unknown values.
```

Comment:
Hmm, Normalize's Go doc says

> Normalize validates and translate the platform to the canonical value.

We need to either change the doc or the implementation regarding the validation part.